noir-projects/aztec-nr/aztec/src/note/note_interface.nr

```diff
@@ -1,21 +1,21 @@
 use crate::context::PrivateContext;
 use dep::protocol_types::address::AztecAddress;
 use dep::protocol_types::traits::Empty;
-pub trait NoteProperties<T> {
-    fn properties() -> T;
-}
 
-pub trait PartialNote<S, F>
-where
-    S: Empty,
-    F: Empty,
-{
-    fn setup_payload() -> S;
-
-    fn finalization_payload() -> F;
+// Autogenerated by the #[note] macro
+// docs:start:note_interfaces
+pub trait NoteInterface {
+    fn get_note_type_id() -> Field;
 }
 
-pub trait NullifiableNote {
+pub trait NoteHashing {
```

Comment:
I'd call this `NoteHash`, following other traits that have verbs eg serialize, equal, etc.

Reply:
Addressed 👍